x-pack/plugins/security_solution/cypress/integration/host_details/risk_tab.spec.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { loginAndWaitForHostDetailsPage } from '../../tasks/login';
+
+import { cleanKibana } from '../../tasks/common';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
+import { TABLE_CELL, TABLE_ROWS } from '../../screens/alerts_details';
+import {
+  navigateToHostRiskDetailTab,
+  openRiskFlyout,
+  waitForTableToLoad,
+} from '../../tasks/host_risk';
+
+describe('risk tab', () => {
+  before(() => {
+    cleanKibana();
+    esArchiverLoad('risky_hosts');
+  });
+
+  after(() => {
+    esArchiverUnload('risky_hosts');
+  });
+
+  it('renders risk tab', () => {
+    loginAndWaitForHostDetailsPage('siem-kibana');
+    navigateToHostRiskDetailTab();
+    waitForTableToLoad();
+
+    cy.get('[data-test-subj="topHostScoreContributors"]')
```

Comment:
Please wrap the locator into a constant and extract it to our screens folder.

Reply:
If you use as a locator `'[data-test-subj="topHostScoreContributors"] .euiTableCellContent__text')` then we can make the test more deterministic using should have.text, following this pattern: `cy.get(RULE_NAME).eq(3).should('have.text', 'Unusual Linux Username');`